tests/antithesis/README.md

```diff
@@ -0,0 +1,78 @@
+# Go-based etcd Client Entrypoint
+
+This directory enables integration of Antithesis with etcd. There are 4 containers running in this system: 3 that make up an etcd cluster (etcd0, etcd1, etcd2) and one that "[makes the system go](https://antithesis.com/docs/getting_started/basic_test_hookup/)" (client).
+
+## Quickstart
+
+### 1. Build and Tag the Docker Image
+
+Run this command from the `antithesis/test-template` directory:
+
+```bash
+docker build . -f Dockerfile.client -t etcd-client:latest
+```
+
+### 2. (Optional) Check the Image Locally
+
+You can verify your new image is built:
+
+```bash
+docker images | grep etcd-client
+```
+
+It should show something like:
+
+```
+etcd-client        latest    <IMAGE_ID>    <DATE>
+```
+
+### 3. Use in Docker Compose
+
+If you have a `docker-compose.yml` that defines your etcd cluster plus this client, make sure the `client` service references `etcd-client:latest`, for example:
+
+```yaml
+services:
+  etcd0:
+    image: docker.io/bitnami/etcd:3.5
```

Comment:
@serathius aside from replacing `bitnami`, do we want to consider using 3.6.x-rc or a more recent 3.5 version so we are testing against something that's more "recent"?

Reply:
For now latest stable is enough. Lets start with 3.5

Reply:
```suggestion
    image: gcr.io/etcd-development/etcd:v3.5.21
```

Reply:
I am assuming that you generated the new log without using the bitnami image anymore, right?

The suggestion is based on the updated image used in `docker-compose.yml`.

Ref: https://github.com/etcd-io/etcd/pull/19739#discussion_r2040744158

Reply:
ditto